astroid/transforms.py

```diff
@@ -32,9 +32,6 @@ def _transform(self, node: NodeNG) -> NodeNG:
         transformed node.
         """
         cls = node.__class__
-        if cls not in self.transforms:
-            # no transform registered for this class of node
-            return node
 
         transforms = self.transforms[cls]
```

Comment:
Is it safe to do that after removing the condition ?

Reply:
Yes, `transforms` is a `defaultdict` so the `if` statement is completely unnecessary.

_Performance benefit is minimal but I found it when looking at this method. Might give us a 0.00001% boost 😄_